lib/themes/dosomething/paraneue_dosomething/paraneue_dosomething.info

```diff
@@ -28,18 +28,21 @@ settings[header_explore_campaigns_subtext] = 'Any cause, anytime, anywhere.'
 settings[footer_social_facebook_enabled]   = 1
 settings[footer_social_facebook_url]       = https://www.facebook.com/dosomething
 settings[footer_social_facebook_alt]       = dosomething on Facebook
-settings[footer_social_twitter_enabled]    = 1
-settings[footer_social_twitter_url]        = https://twitter.com/dosomething
-settings[footer_social_twitter_alt]        = @dosomething on Twitter
+settings[footer_social_instagram_enabled]  = 1
```

Comment:
wait why did we change this- oh it got removed from line 40 in the previous version
wait where did twitter go- oh now its down at line 40

DRUPAL!